test_station.py

```diff
@@ -68,3 +68,24 @@ def test_inconsistent_typical_range_stations_returns_a_list_of_all_inconsistent_
     actual_inconsistent_stations = inconsistent_typical_range_stations(stations)
     expected_inconsistent_stations = [stations[1], stations[2]]
     assert expected_inconsistent_stations == actual_inconsistent_stations
+
+
+def test_relative_water_level():
+
+    s_id = "test-s-id"
+    m_id = "test-m-id"
+    label = "some station"
+    coord = (-2.0, 4.0)
+    river = "River X"
+    town = "My Town"
+    trange1 = (3, 1)
+    s1 = MonitoringStation(s_id, m_id, label, coord, trange1, river, town)
+
+    assert s1.relative_water_level() is None
+
+    trange2 = (-1.6211, 2.0)
+    s2 = MonitoringStation(s_id, m_id, label, coord, trange2, river, town)
```

Comment:
s2.latest_level = 5

Reply:
check that the relative water level is correct, e.g. if range is 1 to 3, and level is 1.4, expected level is 0.2. check if trange is inconsistent in either way that level is None